src/main/resources/ArtifactRules_new.json

```diff
@@ -1,148 +1,151 @@
+    "IdentifyRules": "links | [?type=='IUT'].target",
     "MatchIdRules": {"_id": "%IdentifyRules_objid%"},
-    "ExtractionRules" : "{testCaseExecutions :[{ testCaseTrigeredEventId:meta.id, testCaseTrigeredTime:meta.time, testCase:data.testCase}]}",
+    "ExtractionRules": "{ testCaseTriggeredEventId: meta.id, testCaseTriggeredTime: meta.time, outcome: data.testCase }",
+    "MergeResolverRules": "[ {NONEPATH:NONE}, {testCaseExecutions: [{ testCaseTriggeredEventId: meta.id }]} ]",
     "ArrayOptions": "",
     "HistoryIdentifyRules": "",
     "HistoryExtractionRules": "",
-    "ProcessRules":null,
-    "ProcessFunction" : null
+    "ProcessRules": null,
+    "ProcessFunction": null
   },
   {
-    "TemplateName":"ARTIFACT_1",
-    "Type":"EiffelTestCaseStartedEvent",
+    "TemplateName": "ARTIFACT_1",
+    "Type": "EiffelTestCaseStartedEvent",
     "TypeRule": "meta.type",
     "IdRule": "meta.id",
     "StartEvent": "NO",
-    "IdentifyRules" : "links | [?type=='TEST_CASE_EXECUTION'].target",
+    "IdentifyRules": "links | [?type=='TEST_CASE_EXECUTION'].target",
     "MatchIdRules": {"_id": "%IdentifyRules_objid%"},
-    "ExtractionRules" : "{testCaseExecutions :[{ testCaseStartedEventId:meta.id, testCaseStartedTime:meta.time, testCase:data.testCase}]}",
-    "MergeResolverRules" : "{\"testCaseTrigeredEventId\":%IdentifyRules%}",
+    "ExtractionRules": "{ testCaseStartedEventId: meta.id, testCaseStartedTime: meta.time, outcome: data.testCase }",
+    "MergeResolverRules": "{\"testCaseTriggeredEventId\": %IdentifyRules%}",
     "ArrayOptions": "",
     "HistoryIdentifyRules": "",
     "HistoryExtractionRules": "",
-    "ProcessRules":null,
-    "ProcessFunction" : null
+    "ProcessRules": null,
+    "ProcessFunction": null
   },
   {
-    "TemplateName":"ARTIFACT_1",
-    "Type":"EiffelTestCaseFinishedEvent",
+    "TemplateName": "ARTIFACT_1",
+    "Type": "EiffelTestCaseFinishedEvent",
     "TypeRule": "meta.type",
     "IdRule": "meta.id",
     "StartEvent": "NO",
-    "IdentifyRules" : "links | [?type=='TEST_CASE_EXECUTION'].target",
+    "IdentifyRules": "links | [?type=='TEST_CASE_EXECUTION'].target",
     "MatchIdRules": { "$and": [{"aggregatedObject.testCaseExecutions.testCaseStartedEventId": "%IdentifyRules%"}]},
-    "ExtractionRules" : "{ testCaseFinishEventId:meta.id, testCaseFinishedTime:meta.time, testCase:data.outcome}",
-    "MergeResolverRules" : "{\"testCaseStartedEventId\":%IdentifyRules%}",
+    "ExtractionRules": "{ testCaseFinishEventId:meta.id, testCaseFinishedTime:meta.time, outcome:data.outcome}",
+    "MergeResolverRules": "{\"testCaseStartedEventId\":%IdentifyRules%}",
     "ArrayMergeOptions": "",
     "HistoryIdentifyRules": "",
     "HistoryExtractionRules": "",
-    "ProcessRules":"{testCaseDuration : diff(testCaseExecutions[0].testCaseFinishedTime, testCaseExecutions[0].testCaseStartedTime)}",
-    "ProcessFunction" : "difference"
+    "ProcessRules": null,
```

Comment:
process rules should not be deleted. Only process function. And delete it completly please